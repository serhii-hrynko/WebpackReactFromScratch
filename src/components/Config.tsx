import React from 'react';

import config from '../config';
import styles from '../assets/styles/config.module.css';

export const Config: React.FC = () => {
    return (
        <div className={styles.config}>
            Config<code>{JSON.stringify(config)}</code>
        </div>
    );
};

export default Config;
