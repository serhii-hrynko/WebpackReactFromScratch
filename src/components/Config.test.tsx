import React from 'react';
import { create } from 'react-test-renderer';
import Config from './Config';

describe('Config.tsx', () => {
    test('Config snapshot', () => {
        const component = create(<Config />);

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
