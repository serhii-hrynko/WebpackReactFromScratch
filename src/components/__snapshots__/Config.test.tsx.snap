// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config.tsx Config snapshot 1`] = `
<div
  className="config"
>
  Config
  <code>
    {}
  </code>
</div>
`;
